packages/api/src/controllers/webhook.ts

```diff
@@ -237,6 +234,39 @@ app.patch(
   }
 );
 
+app.post(
+  "/:id/status",
+  authorizer({ anyAdmin: true }),
+  validatePost("webhook-status-payload"),
+  async (req, res) => {
+    const { id } = req.params;
+    const webhook = await db.webhook.get(id);
+    if (!webhook) {
```

Comment:
Gotta check if the webhook is deleted as well
```suggestion
    if (!webhook || webhook.deleted) {
```

Reply:
Actually, maybe this is one of the few APIs we don't want to check that?